test/testinput/bufr_ncep_mtiasi.yaml

```diff
@@ -6,7 +6,7 @@
 observations:
   - obs space:
       name: bufr
-      obsdatain: "./testinput/gdas.t12z.mtiasi.tm00.bufr_d" 
+      obsdatain: "test/testinput/gdas.mtiasi.t00z.20220216.bufr"
```

Comment:
@smaticka  Looks like the path is wrong here. Perhaps ./testinput/gdas.mtiasi.t00z.20220216.buf ?

Reply:
hmm, when I tried that it failed, but maybe I was in the wrong directory. 
I'll try again from `test`

Reply:
When you run the unit tests the working directory is set to  **_\<ioda-converters dir\>_/test**. The paths need to be relative to this directory.